src/zenml/integrations/gcp/orchestrators/vertex_orchestrator.py

```diff
@@ -30,7 +30,17 @@
 """Implementation of the VertexAI orchestrator."""
 
 import os
-from typing import TYPE_CHECKING, Any, ClassVar, Dict, List, Optional, Tuple
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    ClassVar,
+    Dict,
+    List,
+    Literal,
```

Comment:
I don't think we use it ourselves, but it's installed by some intermediate dependency.
Another solution would be to make this an enum, which would enforce similar validation using pydantic.
However I think the literal approach is quite nice, in case of `typing_extensions` I'd just suggest doing this:
```python
from typing_extensions import Literal
```
instead as that already uses `typing.Literal` on python 3.8+ and the imports look cleaner

Reply:
The easiest would be to 
```
from pydantic.typing import Literal
```
I removed the Literal in favor of a Tuple that allows specifying other constraints apart from the accelerator.